compiler/testData/codegen/box/annotations/annotationDefaultValueOfUnsigned.kt

```diff
@@ -0,0 +1,29 @@
+// TARGET_BACKEND: JVM_IR
+// WITH_STDLIB
+// WITH_REFLECT
+// FULL_JDK
+
+annotation class Ann1(val value: UByte = 41u)
+annotation class Ann2(val value: UShort = 42u)
+annotation class Ann3(val value: UInt = 43u)
+annotation class Ann4(val value: ULong = 44u)
+
+@Ann1
+@Ann2
+@Ann3
+@Ann4
+class A
+
+fun box(): String {
+    val default1 = A::class.java.getDeclaredAnnotation(Ann1::class.java).value
```

Comment:
`Class.getDeclaredAnnotation` is missing on JDK6 and older versions of Android, so the corresponding tests (`compiler/tests-different-jdk`, `compiler/android-tests`) are failing. Let's use `Class.getAnnotation` instead